login-form/src/components/ui-components/RebassButton.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import { Button } from "rebass";
+
+export const RebassButton = ({ value }) => {
+  return <Button>{value}</Button>;
+};
```

Comment:
When developing a component it's a good idea to pass user props to the component as well. This way you can pass HTML events and props as well. As an example let's say somebody wants to pass onClick event to the button, in your code you can't do that. But in the suggested code since I've spread the remaining props from button props onClick will be passed to the button.

```suggestion
export const RebassButton = ({ value, ...otherProps }) => {
  return <Button {...otherProps}>{value}</Button>;
};
```